Minor improvements to config handling

Geod24 opened the pull request:
Pretty simple improvements before the big chunk.

Geod24 committed:
Registry: Move CLI argument handling to its own module

Geod24 committed:
Trivial: Shorten names in registry

This makes it a bit easier to write code and is a bit more idiomatic.
Also fixes some style violations.

Geod24 committed:
Config: Use default value for the logging section

Geod24 committed:
Test: Publicly import Config in agora.test.Base

kchulj commented on source/agora/registry/Config.d:
```diff
@@ -0,0 +1,60 @@
+            "Address where the name register server will bind to, defaults to: " ~ CLIArgs.init.bind_address,
+            &args.bind_address,
+        "bind-port|p",
+            "Port where the name register server will bind to, defaults to: " ~ CLIArgs.init.bind_port.to!string,
```
Typo?
```suggestion
            "Port where the name registry server will bind to, defaults to: " ~ CLIArgs.init.bind_port.to!string,
```

Geod24 replied:
Indeed, but since it was already there, I just added a commit to fix it.

kchulj reviewed (approved):
LGTM

Geod24 committed:
Improve user-visible description of registry's CLI options

Besides fixing the 'name register' typo, this takes into account the presence of the DNS server.
